site/src/sphinx/zookeeper.rst

```diff
@@ -0,0 +1,54 @@
+.. _`Apache ZooKeeper`: https://zookeeper.apache.org/
+.. _`com.linecorp.armeria.client.zookeeper`: apidocs/index.html?com/linecorp/armeria/client/zookeeper/package-summary.html
+.. _`com.linecorp.armeria.server.zookeeper`: apidocs/index.html?com/linecorp/armeria/server/zookeeper/package-summary.html
+
+Setting up a ZooKeeper EndpointGroup
+====================================
+You can put your Endpoints information into an zNode in `Apache ZooKeeper`_ Custer, as node value or as node's child value, like this:
```

Comment:
- Custer -> cluster
- as node value -> as a node value
- as node's child value -> as the child values of a node
- like this -> like the following